scapy/modules/p0f.py

```diff
@@ -530,7 +530,7 @@ def addresult(res):
         # XXX are the packets also seen twice on non Linux systems ?
         count=14
         pl = sniff(iface=iface, filter='tcp and port ' + str(port), count = count, timeout=3)
-        map(addresult, map(packet2p0f, pl))
+        [addresult(e) for e in [packet2p0f(p) for p in pl]]
```

Comment:
This list is not used (and is a list of `None` values, since `addresult()` does not return values). I think it would be better to write:
```Python
    for pkt in pl:
        for elt in packet2p0f(pkt):
            addresult(elt)
```